ext/page_log/palm/palm.c

```diff
@@ -114,6 +118,7 @@ typedef struct {
     uint32_t materialization_delay_ms; /* Average length of materialization delay */
     uint64_t last_materialized_lsn;    /* The last materialized LSN (0 if not set) */
     uint32_t verbose;                  /* Verbose level */
+    uint32_t verbose_msg;              /* Send verbose messages to msg callback interface */
```

Comment:
We treat this like a bool. Should we make it one?

Reply:
Fixed in 61ebd03b54